### DISCUSSION TITLE
How can I put something at the bottom of a LayoutSider?

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#2118] [Q&A] [answered]
Hi

I'm trying to add a "footer" with a "New" button at the **bottom** of the LayoutSider. Whatever I have tried doesn´t move the button from the top...

**Is that possible with Blazorise out of the box or do I need to do some CSS "voodoo"?**

Here is what I have been working with (doesn't work)
```xml
<Layout Sider="true">
    <LayoutSider>
        <LayoutSiderContent>
            <Layout>          
                <LayoutFooter>
                    <Button Color="Color.Primary">New</Button>
                </LayoutFooter>
            </Layout>
        </LayoutSiderContent>
    </LayoutSider>
    <... header and content layout ommited> 
</Layout>
```

**Comment by stsrki** [comment, marked as answer]
I think you will need some CSS voodoo magic to make it work. While Layout will help you to quickly build some basic structure, for anything custom the CSS is best option.
